Library/Homebrew/extend/os/linux/gnu_tar.rb

```diff
@@ -0,0 +1,14 @@
+module UnpackStrategy
+  class Tar
+    include UnpackStrategy
+
+    using Magic
+
+    private
+
+    def extract_to_dir(unpack_dir, basename:, extra_params:, verbose:)
+      extra_params.value = "--user=0 --group=0 --no-same-owner"
+      super(unpack_dir, { :basename => basename }, extra_params.value, { :verbose => verbose })
```

Comment:
`basename: basename, extra_params: extra_params, verbose: verbose`

Reply:
brew style told me to use => 😥

Reply:
.rubocop.yml
```
# depends_on a: :b looks weird in formulae.
Style/HashSyntax:
  EnforcedStyle: hash_rockets
  Exclude:
  - '**/Guardfile'
  - '**/cmd/**/*.rb'
  - '**/lib/**/*.rb'
  - '**/spec/**/*.rb'
```

Reply:
@JuPlutonic https://github.com/Homebrew/brew/blob/d1f755959a2f737162bf7f7d6beb4811c16d7702/Library/Homebrew/.rubocop.yml#L124-L126

Reply:
If you run `brew style` without arguments it will check Homebrew/brew and behave correctly. Not sure how you're running it.